contracts/src/v0.8/dev/automation/2_1/interfaces/IKeeperRegistryMaster.sol

```diff
@@ -89,10 +89,10 @@ interface IKeeperRegistryMaster {
   event UpkeepPerformed(
     uint256 indexed id,
     bool indexed success,
-    uint32 checkBlockNumber,
+    uint96 totalPayment,
     uint256 gasUsed,
     uint256 gasOverhead,
-    uint96 totalPayment
+    bytes trigger
```

Comment:
Would it make sense for the struct to contain a trigger type identifier?
Thinking on how to differentiate between a BlockTrigger or a LogTrigger without checking the absence of properties

Reply:
I think we're going to make the trigger type determinable from the upkeepID, so shouldn't be _100% necessary_... But it's probably going to make it easier on atlas if we just include the type explicitly in the event. And since it's a one time change in gas costs, I'm all for it. Good call out.

Reply:
Edit: I sorry I mixed up which event we were talking about... I think for `UpkeepPerformed` we don't need to add the trigger type because the DB should already have that info. I could see it being useful to add to `UpkeepRegistered` - but I'll hold off for now.